### PR TITLE
update afl++ commit id

### PR DESCRIPTION
**Pull request by vanhauser-thc**
this fixes 2 bugs in afl++, better map size detection, better debugging.

### CODE REVIEW

**Review by inferno-chromium — APPROVED** [review]

File: infra/base-images/base-builder/compile_afl [hunk 1]
```diff
@@ -51,7 +51,10 @@ export ASAN_OPTIONS="detect_leaks=0:symbolize=0:detect_odr_violation=0:abort_on_
 rm -f "$OUT/afl_cmplog.txt"
 test $(($RANDOM % 10)) -lt 4 && {
   export AFL_LLVM_CMPLOG=1
-  # We need to notify afl-fuzz to activate CMPLOG
+}
+
+# We need to notify afl-fuzz to activate CMPLOG
+test -n "$AFL_LLVM_CMPLOG" && {
```
Comment (inferno-chromium) [inline infra/base-images/base-builder/compile_afl:57]: This is causing 

/usr/local/bin/compile_afl: line 57: AFL_LLVM_CMPLOG: unbound variable
Building fuzzers failed.

need to define it if it does not run into that branch.

**Comment by inferno-chromium** [comment]
@vanhauser-thc - that didnt fix it since the exception is on the first $ read from that var.

2021-03-16T08:15:37.6718041Z Copying precompiled afl++
2021-03-16T08:15:37.7025039Z /usr/local/bin/compile_afl: line 52: AFL_LLVM_CMPLOG: unbound variable

**Comment by vanhauser-thc** [comment]
@inferno-chromium it is not ready for merging, still working on zeek, I want to have it all working all in one go.

**Comment by inferno-chromium** [comment]
> @inferno-chromium it is not ready for merging, still working on zeek, I want to have it all working all in one go.

Perfect, thanks, will wait then!

**Comment by vanhauser-thc** [comment]
@inferno-chromium is there anything a project can configure that is read in before compile_afl?
or should project specific afl++ setups be done in /src/build.sh ?

**Comment by inferno-chromium** [comment]
> @inferno-chromium is there anything a project can configure that is read in before compile_afl?
> or should project specific afl++ setups be done in /src/build.sh ?

nope, that is called first, see code in https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/compile#L67
compile_ is called first, build.sh call comes much later.

**Comment by vanhauser-thc** [comment]
@inferno-chromium that does not answer my question :) *where* should I put that? in build.sh (I can undo settings I did in compile_afl) or in compile_afl itself (but I think there is no env to identify the target?) or somewhere else entirely?

**Comment by inferno-chromium** [comment]
> @inferno-chromium that does not answer my question :) _where_ should I put that? in build.sh (I can undo settings I did in compile_afl) or in compile_afl itself (but I think there is no env to identify the target?) or somewhere else entirely?

ah my bad, @oliverchang @jonathanmetzman - thoughts on where we should add that hook or anything else that can be hooked into ?

**Comment by vanhauser-thc** [comment]
IMHO as long as there is no `CONFIG` file or something like that it is better to bake it into build.sh

**Comment by vanhauser-thc** [comment]
it is ready to merge - or did I overlook something with unbound envs?

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**